app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/EventsAdapter.java

```diff
@@ -42,7 +43,6 @@ public EventViewHolder(View view) {
             cardView = (CardView) view;
             titleView =  view.findViewById(R.id.events_title);
             imgView = view.findViewById(R.id.events_img);
-            descriptionView =view.findViewById(R.id.events_description);
```

Comment:
Also delete the variable declaration of descriptionView

Reply:
deleted it .Thank you